src/common/components/layouts/CardWidgetLayout.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+    <p-pane-layout class="card-widget-layout">
+        <slot name="title">
+            <p class="title-wrapper">
+                <span class="title">{{ title }}</span>
+                <span v-if="titleExtra" class="title-extra">{{ titleExtra }}</span>
+            </p>
+        </slot>
+        <p class="value-wrapper" :class="{disabled: value === 0.0 || value === 0}">
+            <span v-if="unitType === UNIT_TYPE.CURRENCY" class="value-unit">{{ unit }}</span>
+            <span class="value">{{ value }}</span>
+            <span v-if="unitType === UNIT_TYPE.PERCENT" class="value-unit">%</span>
+        </p>
+        <slot name="description">
+            <span class="description">{{ description }}</span>
+        </slot>
+        <p-divider v-if="showDivider" class="divider" />
+        <slot />
+    </p-pane-layout>
+</template>
+
+<script lang="ts">
+import { PDivider, PPaneLayout } from '@spaceone/design-system';
+import { reactive, toRefs } from '@vue/composition-api';
+import { CURRENCY } from '@/services/billing/cost-management/cost-analysis/lib/config';
+
+const UNIT_TYPE = Object.freeze({
+    PERCENT: 'PERCENT',
+    CURRENCY: 'CURRENCY',
+});
+
+type UNIT_TYPE = keyof typeof UNIT_TYPE;
```

Comment:
```suggestion
type UNIT_TYPE = typeof UNIT_TYPE[keyof typeof UNIT_TYPE];
```

밑에 `unitType` prop의 validator에서 `UNIT_TYPE`을 쓰셨는데, `Object.values(unitType)`에 대한 타입과 비교하려면, 제안드린 것으로 변경되어야 할 것 같아요!